exporting/read_config.c

```diff
@@ -238,13 +244,21 @@ struct engine *read_exporting_config()
         prometheus_exporter_instance->config.update_every =
             prometheus_config_get_number(EXPORTING_UPDATE_EVERY_OPTION_NAME, EXPORTING_UPDATE_EVERY_DEFAULT);
 
-        // wait for backends subsystem to be initialized
-        // TODO: should we still wait here?
-        for (int retries = 0; !global_exporting_source && retries < 1000; retries++)
-            sleep_usec(10000);
+        // ------------------------------------------------------------------------
+        // collect backward compatibility configuration options
```

Comment:
Do we plan to remove backward compatibility at some point?

Reply:
Good question. I would remove the `backend` section from `netdata.conf` completely right in this PR 😄.

Reply:
In fact, we warned our users, waited for a long time, announced that the backends subsystem will be removed exactly to be able to make breaking changes.

We don't import exporting configuration from the `backend` section anyway. I think that we can drop "global" options and `host_tags` as well.